src/appshell/internal/appshellconfiguration.cpp

```diff
@@ -238,3 +270,43 @@ std::string AppShellConfiguration::currentLanguageCode() const
 
     return locale.bcp47Name().toStdString();
 }
+
+mu::io::path AppShellConfiguration::sessionDataPath() const
+{
+    return globalConfiguration()->userAppDataPath() + "/session";
```

Comment:
I think better to add extension (.json ?)

Reply:
It is a directory for session.
The extension for session is .json. See sessionFilePath and 
static const mu::io::path SESSION_FILE("/session.json");